src/tests/services/getVideos.test.js

```diff
@@ -0,0 +1,47 @@
+import { getVideos } from '../../services/getVideos';
+
+describe('Testing on services/getVideos', () => {
+    test('should return array empty when search string is empty', async () => {
+        const videos = await getVideos('');
+        expect(videos.length).toBe(0);
+    });
+
+    test('should return correct array object structure', async () => {
+        /// THIS TEST IS COMMENT BECAUSE CONSUME GOOGLE API KEY AND IT CAN
+        /// MAKE EXCEED THE DAILY QUOTA SO
+        /// IF YOU WANT TO TEST, JUST UNCOMMENT
+        // process.env.REACT_APP_GOOGLE_API_KEY = 'AIzaSyAZYBNRYY1E71H10vLse8JnAo5mO2kE3Ak';
```

Comment:
This is sensitive data, if that's your real api key, I suggest you delete it, remove it from code and create a new one. Do not publish sensitive data ever into github.